src/layout/components/AppMain.vue

```diff
@@ -52,7 +52,7 @@ export default {
 <style lang="scss" scoped>
 .app-main {
   background-color: #f3f3f4;
-  height: 100% !important;
+  height: 100vh !important;
   width: 100%;
   position: relative;
   overflow: auto;
```

Comment:
I'm sorry, but I need more specific details about which code snippets you refer to. Can you please share them? Without the context or the snippet names, I can't evaluate the code differences you're asking me about.